alvr/server_core/src/connection.rs

```diff
@@ -896,10 +897,24 @@ fn connection_pipeline(
                     Err(ConnectionError::TryAgain(_)) => continue,
                     Err(ConnectionError::Other(_)) => return,
                 };
-                let Ok(tracking) = data.get_header() else {
+                let Ok(mut tracking) = data.get_header() else {
                     return;
                 };
 
+                if !streaming_caps.multimodal_protocol {
```

Comment:
Doesn't the client explicitly add them in when multimodal protocol isn't negotiated? Why would we remove them here?

Reply:
Because we go back to multimodal protocol for the ServerCoreContext interface. I know it's a lot of back and forth. but it's worth having a stable client_core and server_core interface at their API level

Reply:
I think the fact that the server is actually multimodal internally should really be documented somewhere, but yea makes sense

Reply:
well yeah... but I'd prefer for this to be the default mode, it will be default in v21 which is coming relatively soon

Reply:
fair enough, that'll need a bit of rewriting then tho, but that's your job ;)